src/chainparams.cpp

```diff
@@ -183,6 +183,7 @@ class CMainParams : public CChainParams
         consensus.ZC_MinMintFee = 1 * CENT;
         consensus.ZC_MinStakeDepth = 200;
         consensus.ZC_TimeStart = 1508214600;        // October 17, 2017 4:30:00 AM
+        consensus.ZC_HeightStart = 863735;
```

Comment:
why this and not use/modify `UPGRADE_ZC`?
The NU was created purely to fulfill the block activation purpose.

Reply:
`UPGRADE_ZC` is a different block height. And sets the block version v4, enabling zc spends.
As explained in the comment on the contextual check, mints were enabled via block-time. We are just replacing it with block height here, not changing behavior.
I wouldn't create a different NU param only for this rule but, in any case, it's a trivial thing that can be addressed in a separate PR if deemed necessary.

Reply:
ok np.

future talk would be if we should remove `MAX_BLOCK_SIZE_LEGACY`, as it's only used for the max sigops constant which is the only reason for keeping the `ZC_TimeStart` after this PR.